topics/contributing/tutorials/updating_tutorial/tutorial.md

```diff
@@ -0,0 +1,166 @@
+
+</div>
+
+<div class="Yes" markdown="1">
+
+> <hands-on-title>Import the workflow</hands-on-title>
+>
+> 1. **Navigate to your target {% icon hands_on %} Hands-on tutorial**
+> 2. **Select the {% icon workflow %} workflow** from the header
+> 3. **Import the workflow** to your server
+> 4. Go to the **Workflow** menu and select {% icon galaxy-wf-edit %} **Edit workflow**
+{: .hands_on}
+
+</div>
+
+# Phase 2: Check that nobody else is working on this
+
+It's always a good idea to check, just in case!
+
+You can:
+ - Send a message to the [GTN Matrix Channel](https://matrix.to/#/#Galaxy-Training-Network_Lobby:gitter.im) is your quickest way forward.
+ - Search through the [GTN Github Repository](https://github.com/galaxyproject/training-material) for existing draft Pull Requests.
+ - Check with [individual communities](https://galaxyproject.org/community/sig/), who may have their own method of tracking. For example, the [🖖🏾Single-cell & sPatial Omics Community](https://galaxyproject.org/community/sig/singlecell/) have a shared [Click-Up board](https://sharing.clickup.com/9015477668/b/h/5-90152810734-2/557452707486fef) at the time of writing.
+
+# Phase 3: Update the workflow
+
+> <hands-on-title>Update the workflow</hands-on-title>
+>
+> 1. Select {% icon galaxy-wf-options %} **Workflow Options** from the **Workflow Editor**
+> 2. Select {% icon upgrade_workflow %} **Upgrade workflow** to automatically update the tools.
+> 3. Address any issues that may arise.
+> 4. **Save** the workflow.
+{: .hands_on}
+
+
+# Phase 4: Test & fix the workflow
+
+> <hands-on-title>Test the workflow</hands-on-title>
+>
+> 1. Import the input datasets from the tutorial you are upgrading (follow the instructions on the tutorial itself for this).
+> 2. Run your updated workflow on the input datasets.
+> 3. Address any issues that may arise, and **NOTE DOWN** all changes.
+> 4. Ensure the workflow meets workflow best practices using the {% icon galaxy-wf-best-practices %} **Best Practices** button and add your name as a contributor, if not there already.
+>
+>    {% snippet faqs/galaxy/workflows_best_practices.md %}
+>
+> 5. Add yourself as an author of the workflow.
+> 6. **Save** the workflow.
+> 7. Run your updated workflow on the input datasets in a fresh history.
+> 7. **Save** this history as an answer key history & make your history publicly available in Published Histories.
+>
+>    {% snippet faqs/galaxy/histories_sharing.md %}
+>
+{: .hands_on}
+
+# Phase 5: Update the tutorial
+
+> <warning-title>If you run out of time</warning-title>
+> You have already completed a large chunk of work (updated workflow + answer key history) to get here, and we don't want to lose it!
+> Updating the tutorial to match the updated workflow can be a separate contribution. So if you get to this point and run out of time, please:
+> 1. Create an **issue** on the GTN Github Repo and include shareable links to your updated workflow & answer key history
```

Comment:
```suggestion
> 1. Create an **issue** on the GTN Github repositoty and include shareable links to your updated workflow & answer key history
```

Reply:
I am not sure to understand what is "answer key history"

Reply:
added in labels with `` throughout to emphasise this